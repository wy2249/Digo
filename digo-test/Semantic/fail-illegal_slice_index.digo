func digo_main() void {
  a := []int {1,2,3,4}
  b := a[1.0]
}
