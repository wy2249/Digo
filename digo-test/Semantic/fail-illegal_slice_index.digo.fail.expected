Fatal error: exception Failure("illgal slice index: non-integer index")
